src/components/SeatMap/SeatMap.test.js

```diff
@@ -0,0 +1,252 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { JetsSeatMap } from './SeatMap';
+import { CONFIG_MOCK } from '../Demo/constants';
+import { flightDetails } from './__fixtures__/seatMapApiGetPlaneFeatures';
+import { finalDeck } from './__fixtures__/SeatMapService';
+import { paramsData } from '../TooltipGlobal/__fixtures__';
+import { row, seat } from './__fixtures__/seatMapApiPostDataResponse';
+import { JetsSeatMapService } from './service';
+
+jest.mock('./service');
+
+const setup = ({ flight, availability, passengers, currentDeckIndex, seatJumpTo, config, onSeatMapInited }) => {
```

Comment:
Probably we can remove the seatJumpTo parameter from here as in the tests always null is passed